platform-dom/src/main/java/com/softicar/platform/dom/elements/input/auto/DomAutoCompleteInput.java

```diff
@@ -35,9 +36,15 @@
 	protected final DomBar inputBar;
 	private final boolean sloppyAmbiguityCheck;
 	private final DomAutoCompleteInputFilterDisplay filterDisplay;
-	private final DomTextInput inputField;
+	private final DomAutoCompleteBackdrop backdrop;
+	private final DomAutoCompletePopup<T> popup;
+	private final DomAutoCompleteInputField inputField;
 	private final IDomAutoCompleteInputConfiguration configuration;
 	private final Collection<INullaryVoidFunction> inputConstraintRefreshCallbacks;
+	private boolean showBackdrop;
+	private DomAutoCompleteIndicator indicator;
+	private T committedValue;
+	private boolean focus;
```

Comment:
should probably be renamed to `focussed`

Reply:
renamed to hasFocus